astrokat/scans.py

```diff
@@ -174,18 +335,21 @@ def return_scan(session, target, nd_period=None, lead_time=None, **kwargs):
     """
     # set up 2way scan
     user_logger.info("Forward scan over target")
-    target_visible = forwardscan(session,
-                                 target,
-                                 nd_period=nd_period,
-                                 lead_time=lead_time,
-                                 **kwargs)
+    target_visible = scan(session,
+                          target,
+                          nd_period=nd_period,
+                          lead_time=lead_time,
+                          **kwargs)
 
     user_logger.info("Reverse scan over target")
-    target_visible += reversescan(session,
-                                  target,
-                                  nd_period=nd_period,
-                                  lead_time=lead_time,
-                                  **kwargs)
+    returnscan = dict(kwargs)
+    returnscan["start"] = kwargs["end"]
+    returnscan["end"] = kwargs["start"]
+    target_visible += scan(session,
```

Comment:
Here is another `+=` used with booleans.   Should we be using `&=` instead? 

```python
In [1]: x = True

In [2]: x &= True

In [3]: x
Out[3]: True

In [4]: x &= False

In [5]: x
Out[5]: False

In [6]: x += True

In [7]: x
Out[7]: 1

In [8]: x += False

In [9]: x
Out[9]: 1

In [10]: x += True

In [11]: x
Out[11]: 2
```

Reply:
Done